Makefile

```diff
@@ -21,15 +21,9 @@ migrate:
 	poetry run python manage.py makemigrations
 	poetry run python manage.py migrate
 
-colon := :
-
-runGunicorn:
-	poetry run gunicorn -b 0.0.0.0$(colon)8000 answerking.wsgi$(colon)application
-
-waitAndMigrate:
```

Comment:
need to rebase as these changes should be accepted

Reply:
has been rebased, the escape key '\' works the same way so going to stay with that